views/sponsors.php

```diff
@@ -27,6 +29,124 @@
     <!-- Navbar -->
     <?php include('../views/includes/navbar.html'); ?>
 
+    <!--Body -->
+    <div class = "allcontent">
+ <h1 align = "center" style = "margin-top: 50px">
```

Comment:
instead of inlining the styling, can you just make it so that all h1 tags have 50px above them with CSS?
```
h1 {
  margin-top: 50px;
}
```